src/database/all.sql

```diff
@@ -3,15 +3,43 @@
 -- --------------------------- 
   
   
+-- ---------------------------- 
+-- From: 000_drop_all.sql 
+SET FOREIGN_KEY_CHECKS = 0;
+
+DROP TABLE IF EXISTS address;
+DROP TABLE IF EXISTS associated_service;
+DROP TABLE IF EXISTS authorisation;
+DROP TABLE IF EXISTS bid_status;
+DROP TABLE IF EXISTS billing;
+DROP TABLE IF EXISTS billing_type;
+DROP TABLE IF EXISTS client;
+DROP TABLE IF EXISTS professional;
+DROP TABLE IF EXISTS request;
+DROP TABLE IF EXISTS request_bid;
+DROP TABLE IF EXISTS request_status;
+DROP TABLE IF EXISTS request_subscription;
+DROP TABLE IF EXISTS request_transaction;
+DROP TABLE IF EXISTS security_question;
+DROP TABLE IF EXISTS service;
+DROP TABLE IF EXISTS session;
+DROP TABLE IF EXISTS subscription;
+DROP TABLE IF EXISTS transaction;
+DROP TABLE IF EXISTS transaction_status;
+DROP TABLE IF EXISTS user;
+DROP TABLE IF EXISTS user_question;
+
+SET FOREIGN_KEY_CHECKS = 1;  
+  
 -- ---------------------------- 
 -- From: 010_user.sql 
 CREATE TABLE user (
      user_id            MEDIUMINT NOT NULL AUTO_INCREMENT,
-     first_name         CHAR(100) NOT NULL,
-     last_name          CHAR(100) NOT NULL,
-     email_address      CHAR(100) NOT NULL,
-     mobile             CHAR(100) NOT NULL,
-     password           CHAR(100) NOT NULL,
+     first_name         VARCHAR(100) NOT NULL,
+     last_name          VARCHAR(100) NOT NULL,
+     email_address      VARCHAR(200) NOT NULL,
+     mobile             VARCHAR(20) NOT NULL,
+     password           VARCHAR(20) NOT NULL,
```

Comment:
Please increase from VARCHAR(20) to VARCHAR(32) minimum due to Front-End hashing passwords. This is already implemented on the Front-End will be easier to increase the size of field.